main.tex

```diff
@@ -883,15 +883,20 @@ \subsection{Modeling}
 % Nadia, Lim
 % Tom R. -- unit support
 
-The \package{astropy.modeling} subpackage provides a framework for representing
-analytical models and performing model evaluation and parameter fitting. Models and
-fitters are independent of each other: a model can be fit with different
+Tha \package{astropy.modeling} subpackage provides a framework for representing
+analytical models and performing model evaluation and parameter fitting.
+The main motivation for it was creating a framework
```

Comment:
A few grammar things: "The main motivation for this functionality was to create a framework that allows arbitrary combination of models to..."